### ISSUE TITLE
channel closed: Error: channel closed

### ISSUE THREAD

**Issue by nicollaas94** [HvyIndustries/crane#205]
```
shell.ts:541channel closed: Error: channel closed
    at ChildProcess.target.send (internal/child_process.js:540:16)
    at IPCMessageWriter.write (C:\Users\Nicollas\.vscode\extensions\HvyIndustries.crane-0.2.2\node_modules\vscode-jsonrpc\lib\messageWriter.js:34:22)
    at C:\Users\Nicollas\.vscode\extensions\HvyIndustries.crane-0.2.2\node_modules\vscode-jsonrpc\lib\main.js:195:31
    at Object.sendRequest (C:\Users\Nicollas\.vscode\extensions\HvyIndustries.crane-0.2.2\node_modules\vscode-jsonrpc\lib\main.js:186:20)
    at Object.sendRequest (C:\Users\Nicollas\.vscode\extensions\HvyIndustries.crane-0.2.2\node_modules\vscode-languageclient\lib\main.js:43:66)
    at LanguageClient.doSendRequest (C:\Users\Nicollas\.vscode\extensions\HvyIndustries.crane-0.2.2\node_modules\vscode-languageclient\lib\main.js:167:31)
    at C:\Users\Nicollas\.vscode\extensions\HvyIndustries.crane-0.2.2\node_modules\vscode-languageclient\lib\main.js:160:30
    at process._tickCallback (internal/process/next_tick.js:103:7)e.onUnexpectedError @ shell.ts:541
2extensionHost.ts:293[Extension Host] rejected promise not handled within 1 second
```

**Comment by nevadascout** [comment]
Fixed in v0.3.0

**Comment by DefectBuster** [comment]
Faced the same issue, and I terminated the application terminal with [Ctrl+c] and rerun the command [ng serve] and it worked for me.
